Frends.PDF.Read/README.md

```
@@ -14,7 +14,7 @@ You can install the Task via Frends UI Task View.
 
 ### Clone a copy of the repository
 
-`git clone https://github.com/FrendsPlatform/Frends.PDF.git`
+`git clone https://github.com/FrendsPlatform/Frends.Pdf.git `
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)